rts/Lua/LuaSyncedRead.cpp

```diff
@@ -125,6 +125,7 @@ bool LuaSyncedRead::PushEntries(lua_State* L)
 	REGISTER_LUA_CFUNC(GetPlayerRulesParams);
 
 	REGISTER_LUA_CFUNC(GetMapOptions);
```

Comment:
Would be good to also have singular `GetMapOption`, should be a matter of copy paste

Reply:
Added